website/blog/2022-06-11-2.7.0.md

```diff
@@ -0,0 +1,513 @@
+
+test.describe.parallel("does something really long and complicated so I have to write a very long name for the test", () => {});
+
+test.describe.parallel.only("does something really long and complicated so I have to write a very long name for the testThis is a very", () => {});
+
+test.describe.serial("does something really long and complicated so I have to write a very long name for the test", () => {});
+
+test.describe.serial.only("does something really long and complicated so I have to write a very long name for the test", () => {});
+
+// Prettier 2.6
+test.step(
+  "does something really long and complicated so I have to write a very long name for the test",
+  () => {}
+);
+
+test.describe(
+  "does something really long and complicated so I have to write a very long name for the test",
+  () => {}
+);
+
+test.describe.only(
+  "does something really long and complicated so I have to write a very long name for the test",
+  () => {}
+);
+
+test.describe.parallel(
+  "does something really long and complicated so I have to write a very long name for the test",
+  () => {}
+);
+
+test.describe.parallel.only(
+  "does something really long and complicated so I have to write a very long name for the testThis is a very",
+  () => {}
+);
+
+test.describe.serial(
+  "does something really long and complicated so I have to write a very long name for the test",
+  () => {}
+);
+
+test.describe.serial.only(
+  "does something really long and complicated so I have to write a very long name for the test",
+  () => {}
+);
+
+// Prettier 2.7
+test.step("does something really long and complicated so I have to write a very long name for the test", () => {});
+
+test.describe("does something really long and complicated so I have to write a very long name for the test", () => {});
+
+test.describe
+  .only("does something really long and complicated so I have to write a very long name for the test", () => {});
+
+test.describe
+  .parallel("does something really long and complicated so I have to write a very long name for the test", () => {});
+
+test.describe.parallel
+  .only("does something really long and complicated so I have to write a very long name for the testThis is a very", () => {});
+
+test.describe
+  .serial("does something really long and complicated so I have to write a very long name for the test", () => {});
+
+test.describe.serial
+  .only("does something really long and complicated so I have to write a very long name for the test", () => {});
+```
+
+#### Fix comment formats in ([#12860](https://github.com/prettier/prettier/pull/12860) by [@HosokawaR](https://github.com/HosokawaR))
+
+This change fixes the comment format in `exports` to align with the comment format in `import`.
+
+Although this change does not affect the comments format in `import`, follows change log contains examples of comments in `import` for reference.
```

Comment:
```suggestion
Although this change does not affect the comments format in `import`, the following changeset contains examples of comments in `import` for reference.
```

(Skeptical about either "change log" or "changeset" being the right word, but nothing better on my mind)